server-spi-private/src/main/java/org/keycloak/events/EventType.java

```diff
@@ -36,7 +36,8 @@
     CLIENT_LOGIN_ERROR(true),
 
     REFRESH_TOKEN(false),
-    REFRESH_TOKEN_ERROR(false),
```

Comment:
Nitpick, but it helps to have a more concise set of changes and make review easier. Could you revert this change?

Reply:
Sure, we reverted all formatting changes we found. Hope it's fine like this.